### ISSUE TITLE
Allow multiple --matches and --not-matches patterns

### ISSUE THREAD

**Issue by eikek** [docspell/dsc#135]
They could all be tried and first match wins.

**Comment by ScarletStar** [comment]
You mean as an OR operation?

I think something like this: 
(matches OR matches OR ....) AND (not-matches OR not-matches OR ...)

**Comment by eikek** [comment]
Yes that's what I thought. If you specify multiple patterns in `--matches` at least one must match, same for `--not-matches`. But then both results are combined via AND. Or did you think of something else?

**Comment by ScarletStar** [comment]
That's  all I can think of for now 🤔
